spec/models/course_user_spec.rb

```diff
@@ -162,6 +162,15 @@
       end
     end
 
+    describe '#manager_or_owner?' do
+      it 'returns true if the role is manager or owner' do
+        expect(student.manager_or_owner?).to be_falsey
```

Comment:
Also possible, falsey and truthy catch slightly more values of true and false in the cases where we use them for conditionals.